src/modules/support/login.ts

```diff
@@ -60,17 +62,18 @@ const assertToken = (raw: string): void => {
 }
 
 const saveSupportCredentials = (account: string, token: string): void => {
-  saveAccount(account)
-  saveWorkspace('master')
-  saveToken(token)
+  const session = SessionManager.getSingleton()
+  session.DEPRECATEDchangeAccount(account)
```

Comment:
Do you really want the deprecated one?

Reply:
In this case I don't really know how `support login` works, nor how to test it - I created the deprecated one just for this. In a future PR we would want to change it

Reply:
This should use `session.login`, providing a login method, implemented here:
https://github.com/vtex/toolbelt/blob/e3abd57ab55eb80d648f7f53d20e1c6054090b6a/src/lib/auth/AuthProviders/AuthProviders.ts#L5-L9